src/components/CompareResults/Retrigger/RetriggerButton.tsx

```diff
@@ -156,14 +178,28 @@ function RetriggerButton({ result }: RetriggerButtonProps) {
 
   return (
     <>
-      <IconButton
-        title={Strings.components.revisionRow.title.retriggerJobs}
-        color='primary'
-        size='small'
-        onClick={() => void onRetriggerButtonClick()}
-      >
-        <RefreshOutlinedIcon />
-      </IconButton>
+      {variant === 'text' ? (
+        <Button
+          title={Strings.components.revisionRow.title.retriggerJobs}
+          className={styles.retrigger}
+          color='primary'
+          variant={'text'}
+          size='small'
```

Comment:
Once all styles are removed, the small size doesn't look right, so please remove it.